app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands = command.split(" ")
```

Comment:
" " is default value for split
```suggestion
    commands = command.split()
```